### PR TITLE
fix: unorphaned `avm/res/compute/image`

### PR DESCRIPTION
**Pull request by eriqua**
## Description

Unorphaned `avm/res/compute/image`

### CODE REVIEW

File: avm/res/compute/image/README.md [hunk 1]
```diff
@@ -1,10 +1,5 @@
 # Images `[Microsoft.Compute/images]`
 
-> ⚠️THIS MODULE IS CURRENTLY ORPHANED.⚠️
```
Comment (eriqua) [inline avm/res/compute/image/README.md:3]: Waiting for @tony-box membership to relevant avm groups

**Review by AlexanderSehr — APPROVED** [review]

File: avm/res/compute/image/README.md (hunk 1 above)
Comment (ChrisSidebotham) [reply avm/res/compute/image/README.md:3]: @eriqua - I believe the decision was to not block this in BRM due to group membership and pick up another task in AVM Governance to manage / track ownership?

Comment (matebarabas) [reply avm/res/compute/image/README.md:3]: Note that the group membership has been fixed by @segraef, confirmed here: https://github.com/Azure/Azure-Verified-Modules/issues/568

Comment (eriqua) [reply avm/res/compute/image/README.md:3]: @tony-box this is ready to be approved as well :sunglasses:

**Pull request merged**